dask/dataframe/io/delta/utils.py

```diff
@@ -0,0 +1,82 @@
+import json
+from typing import Union
+
+import pyarrow as pa
```

Comment:
I see that everything here is pyarrow specific

Reply:
yeah, I am also a bit worried here, because currently reading delta files are achieved only through pyarrow dataset, not able to achieve the same using fastparquet. that's why I am wondering whether we can fix the engine as pyarrow only for reading delta files as of now?